### PR TITLE
feat: add aruba silverpeak parser

### PR DESCRIPTION
**Pull request by wojtekzyla**
Add a parser for Aruba SilverPeak requested in #2529

### CODE REVIEW

File: docs/sources/vendor/Aruba/silverpeak.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Clearpass
```
Comment (mstopa-splunk) [inline docs/sources/vendor/Aruba/silverpeak.md:1]: Please update the header

File: docs/sources/vendor/Aruba/silverpeak.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+## Links
+
+| Ref            | Link                                                                                                    |
+|----------------|---------------------------------------------------------------------------------------------------------|
+
+## Sourcetypes
+
+| sourcetype       | notes                                                                                                   |
+|------------------|---------------------------------------------------------------------------------------------------------|
+| aruba:silverpeak | Dynamically  Created                                                                                    |
```
Comment (mstopa-splunk) [inline docs/sources/vendor/Aruba/silverpeak.md:17]: We can remove `Dynamically  Created` from this doc

File: docs/sources/vendor/Aruba/silverpeak.md (hunk 1 above)
Comment (wojtekzyla) [reply docs/sources/vendor/Aruba/silverpeak.md]: Done

File: docs/sources/vendor/Aruba/silverpeak.md (hunk 2 above)
Comment (wojtekzyla) [reply docs/sources/vendor/Aruba/silverpeak.md]: Done

**Review by rjha-splunk — APPROVED** [review]

**Pull request merged**